include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Failed to parse {path}, expected a top-level object.");
-
-}
+    DECLARE_MESSAGE(FailedToParseSerializedManifest,
+                    (msg::path),
+                    "",
+                    "Failed to parse serialized manifest file of {path} Please open an issue at "
+                    "https://github.com/microsoft/vcpkg, with the following output:");
+    DECLARE_MESSAGE(MismatchedSerializedManifestSCF,
+                    (),
+                    "",
+                    "The serialized manifest SCF was different from the original SCF. Please open an issue at "
```

Comment:
I don't think the translator is going to know what this SCF thing means. I think this should just use `InternalErrorMessageContact` ?

Reply:
SCF stands for "SourceControlFile" -- it really just means "manifest". I think this message should drop SCF:

```suggestion
                    "The serialized manifest was different from the original. Please open an issue at "
```